fluent-syntax/src/parser.js

```diff
@@ -394,6 +395,17 @@ export default class FluentParser {
     return new AST.NumberLiteral(num);
   }
 
+  getValue(ps) {
+    if (ps.isPeekNextLineVariantStart()) {
```

Comment:
This doesn't work because `isPeekNextLineVariantStart` expect to be called at the end of line. We need to first check for the opening `{` here.